src/stores/balanceListStore.ts

```diff
@@ -28,6 +28,11 @@ class BalanceListStore extends BaseStore {
       .reduce(add, 0);
   }
 
+  @computed
+  get checkAnyFunds() {
```

Comment:
weird naming, my first impressionwas it is a function? not computed field

Reply:
plus it does not do what id declares? i just don't see any **check** in the impl